### PR TITLE
add my card information

### PR DESCRIPTION
**Pull request by arnab9593**
all information added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,34 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="https://github.com/donnemartin/system-design-primer" target="_blank" title="GitHub">System Design</a>
+              </li>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="FreeCodeCamp">Fundamentals of Coding</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________ Marek card END ________  -->
```
Comment (alsonick) [inline index.html:144]: This comment should include your name.

**Review by alsonick — CHANGES_REQUESTED** [review]
Can you update this comment with your name instead, thanks.

File: index.html (hunk 1 above)
Comment (arnab9593) [reply index.html]: changes done

**Review by CandidDeer — APPROVED** [review]

**Comment by CandidDeer** [comment]
Thanks for your contribution. I have merged your PR and your card will be live in a moment.

**Pull request merged**